app/routes/_index.tsx

```diff
@@ -1,23 +1,175 @@
+    let captioningMinutes =
+      formState.averageProgramsPerMonth *
+      formState.averageLengthOfProgramsInHours *
+      60;
+    setCaptionMinutes(captioningMinutes);
+  };
+
   return (
-    <div>
-      <h1 className="text-3xl font-bold underline">Hello world!</h1>
-      {/*  
-        // Use the form below if you wan to go remix style, or just use fetch in an event handler. 
-      */}
-      <form action="/api/leads" method="POST">
-        <label htmlFor="name">Name</label>
-        <input id="name" name="name" placeholder="Pat Smith" />
-      </form>
+    <div className="app-container mx-auto index mt-12 px-5">
+      <h1 className="text-5xl uppercase text-center	heading mb-5">
+        Captioning Calculator
+      </h1>
+      <div className="grid grid-cols-1 content-center md:grid-cols-2">
+        <div className="grid content-center text-center	md:text-left">
+          <h2 className="text-3xl mb-3">
+            How many captioning minutes will you need?
+          </h2>
+          <p className="text-lg">
+            The Cablecast Captioning Calculator is a tool to understand how many
+            Cablecast Captioning Minutes you will need for a year of
+            programming.
+          </p>
+        </div>
+        <div>
+          <img src="cc-decorative-img.png" alt="" className="decorative-img" />
+        </div>
+      </div>
+      <div className="form-container rounded-3xl items-center mb-10">
+        {hasFormSubmit ? (
+          <>
+            <h2 className="text-3xl text-center pt-10 mt-5">
+              You will need approximately
+            </h2>
+            <div className="flex items-center">
+              <p className="minutes-needed">{captionMinutes} minutes </p>
+            </div>
+            <h2 className="text-3xl text-center pb-10">
+              {" "}
+              Of Closed Captioning for 1 year
+            </h2>
+          </>
+        ) : (
+          <form className="flex flex-col text-2xl mt-5" onSubmit={handleSubmit}>
+            <FormInput
+              type="text"
+              label="Name"
+              name="name"
+              id="name"
+              value={formState.name}
+              handleInputChange={handleInputChange}
+              image="name.png"
+              required
+            />
+            <FormInput
+              type="email"
+              label="Email"
+              name="email"
+              id="email"
+              value={formState.email}
+              handleInputChange={handleInputChange}
+              image="email.png"
+              required
+            />
+            <FormInput
+              type="number"
+              label="Average Number of Programs Per Month"
+              name="averageProgramsPerMonth"
+              id="averageProgramsPerMonth"
+              value={formState.averageProgramsPerMonth}
+              handleInputChange={handleInputChange}
+              image="number.png"
+              min="0"
```

Comment:
Nice call on adding validation to these form inputs. Would probably put 1 here as if you have no programs a month the calculator isn't going to spit out a very useful answer :)